src/main/resources/application-local-in-docker.properties

```diff
@@ -0,0 +1,13 @@
+#logging.level.root=DEBUG
```

Comment:
we can keep debug level in local ?

Reply:
yes, but i think it's too much information, we can activate it manually when we need it though

Reply:
ok